src/actions/postActions.js

```diff
@@ -0,0 +1,13 @@
+export const CREATE_POST = 'CREATE_POST';
+
+export const createPost = post => ({
+  type: CREATE_POST,
+  payload: post
+});
+
+export const DELETE_POST = 'DELETE_POST';
+
+export const deletePost = title => ({
+  type: DELETE_POST,
+  payload: title
```

Comment:
if you have posts with the same title, it will delete all of them. Not a big deal but something to think about.